extension.ts

```diff
@@ -70,9 +70,9 @@ namespace hourOfCode {
     //% weight=80
     export function agentDetectDryBrush(dir: SixDirection) {
         let currentTarget = agent.inspect(AgentInspection.Block, dir)
-        return currentTarget == tallHazard || currentTarget == shortHazard
+        return tallHazards.indexOf(currentTarget) != -1 || shortHazards.indexOf(currentTarget) != -1
```

Comment:
This might not be super needed, but could you just re-use the `agentDectectDryFern` and `agentDetectDryGrass` functions here?

Reply:
You can do this in most of the other places, but I don't feel too strongly about it if you'd like to keep your changes as is.

Reply:
I considered this, but I didn't want to trigger 2 separate calls into agent.inspect() which could slow things down.

If I have to go back and adjust it again, I'll probably make a helper function that just takes in a number and does the indexOf check, but for now I think it's simplest to leave it be.

Reply:
Sounds good!